src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -43,17 +56,17 @@ Sometimes, you need some setup code, or other things that would distract
 from your example, but are important to make the tests work. Consider
 an example block that looks like this:
 
-```text
-/// Some documentation.
-# fn foo() {}
-```
+   ```text
+   /// Some documentation.
+   # fn foo() {}
+   ```
```

Comment:
But that's not the point. The point is to show what happens when you hide a line with `#`, not to highlight the text differently.